src/otx/core/model/keypoint_detection.py

```diff
@@ -10,7 +10,6 @@
 
 import torch
 
-from otx.algo.keypoint_detection.utils.data_sample import PoseDataSample
 from otx.algo.utils.mmengine_utils import load_checkpoint
```

Comment:
This also looks like calling `otx.algo` from inside `otx.core`, like my previous comment. How about changing this to calling it from inside a function?

Reply:
The same function is also used for detection and instance_segmentation. Can we take a look at the another PR?